src/Orchard.Web/Modules/Orchard.Autoroute/Settings/AutorouteSettings.cs

```diff
@@ -39,18 +45,58 @@ public class AutorouteSettings {
                 return _patterns;
             }
 
-            set { 
+            set {
                 _patterns = value;
                 PatternDefinitions = new JavaScriptSerializer().Serialize(_patterns.ToArray());

```

Comment:
We know it's my fault, but please fix it by using the serialization service. Or Json.net directly.